### PR TITLE
Tweaks helbital's more unique effects a bit, and buffs the chem in general

### PR DESCRIPTION
**Pull request by ATH1909**
## About The Pull Request

The chance per tick of helbital's special "janken" effect occurring is now 1/1000, instead of one in a million.

Helbital's "janken" effect no longer blinds you until either the chem leaves your system or 10 MINUTES pass when it triggers (the blindness caused by a helbital OD is still there, though).

Helbital's "janken" effect now rolls its chance to occur each chem tick while you're in crit, instead of while you have brute damage.

Helbital's brute healing formula has been tweaked to be less JANKy (heh), and it also (approximately) heals an extra 0.5 points of brute damage per tick over what it healed before this PR.

Helbital now deals no oxyloss damage to you while you're in soft crit.

You now get a special message indicating how helbital's OD effect is blinding you and dealing burn damage to you when the curse it gets applied.

Tying or winning a game of RPS with the malevolent spirits from its "janken" effect (which causes those spirits to "disappear" and "fade away", respectively) now clears the curse from helbital's OD effect.

Update: Also, epic gamers can no longer just inject themselves with 1000u of helbital, then wait out the curse from its OD effect. You also can't cure your other curses with a 0.1u dose helbital anymore; you gotta OD on it first. Also, the stuff that used to be in on_mob_delete() is now in on_mob_end_metabolize().

## Why It's Good For The Game

Getting helbital's "janken" effect to trigger on a chem tick shouldn't be as likely to happen as winning a pulse rifle from a single game at an arcade machine (which is also a one in a million chance), especially given the extra conditions that are attached to when that janken effect can appear and how rarely people actually use/bother to make helbital.

## Changelog
:cl: ATHATH
balance: The brute healing rate of helbital has been increased.
balance: Helbital no longer deals oxyloss damage to you while you're in soft crit.
balance: Helbital's "janken" effect can no longer trigger while you're conscious, but the chance for it to trigger has been GREATLY increased (to 1/1000 each chem tick, up from one in a million each chem tick), and you no longer need brute damage (before helbital heals you that chem tick) for it to have a chance to trigger.
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
image

**Comment by ATH1909** [comment]
Update: Also, epic gamers can no longer just inject themselves with 1000u of helbital, then wait out the curse from its OD effect. You also can't cure your other curses with a 0.1u dose helbital anymore; you gotta OD on it first. Also, the stuff that used to be in on_mob_delete() is now in on_mob_end_metabolize().

**Comment by Shadowflame909** [comment]
The heck is RPS, and why does one of the most unique and most interesting to use healers come with a curse?

better then "haha ur lungs are now dead"

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 1]
```diff
@@ -13,34 +13,22 @@
...
-	if(good_kind_of_healing && !reaping && prob(0.0001)) //janken with the grim reaper!
+	if(!(M.stat)) //deals oxyloss to you if you're conscious
+		M.adjustOxyLoss(2, TRUE)
+	M.adjustBruteLoss(-(round(death_is_coming/50,0.1)+0.5), FALSE) //heals you more the closer you are to death
+
+	if(M.stat && !reaping && prob(0.1)) //janken with the grim reaper!
 		reaping = TRUE
 		var/list/RockPaperScissors = list("rock" = "paper", "paper" = "scissors", "scissors" = "rock") //choice = loses to
-		if(M.apply_status_effect(/datum/status_effect/necropolis_curse,CURSE_BLINDING))
-			helbent = TRUE
```
[the remark this answers is not in the document]
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm:40]: It shouldn't be applying unless you've rolled for the victory royale anywho.

**Comment by ExcessiveUseOfCobblestone** [comment]
The RPS being compared to a pulse rifle is a bit misleading because yes, if you took 5 rolls of helbital and 5 rolls of pulse rifle, you'd have the same probability. The catch is helbital rolls every life and pulse rifle takes several minutes to even roll.

I have no plans on making this chance smaller, it is suppose to practically never happen.

**Comment by HiKewne** [comment]
A poison that gives you a fate literally worse than death by just overdosing you, and to get cured you have to OD on it again, be in crit _and_ wait until you proc the 1/1000 chances of you having a chance of either getting rid of being permanently blind or being dusted

Now this is epic

**Comment by ATH1909** [comment]
> A poison that gives you a fate literally worse than death by just overdosing you, and to get cured you have to OD on it again, be in crit _and_ wait until you proc the 1/1000 chances of you having a chance of either getting rid of being permanently blind or being dusted
> 
> Now this is epic

No? If you OD, you get cured of the curse if it leaves your system.

**Comment by ATH1909** [comment]
> now, this is just a literal poison, that causes a permanent brain-trauma that can only be cured by critting yourself with a 1/3 chance of perma-death. maybe 1/2 if there's no draw option.

Or you can just purge the helbital from your system, which instantly removes the curse (which ISN'T a brian trauma, by the way).

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (ATH1909) [reply code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm:40]: Rolling for the janken effect now happens more often.

You can only roll the janken effect while not conscious, so we don't really need to blind the person who rolled it and risk weird edge cases happening.

Also, it doesn't look like rolling a tie would have removed this blindness curse before this PR.

[the remark this answers is not in the document]
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm:40]: - I would like that reverted.

- Even if you're unconscious you'll still be able to roll hence why it only has qdel checks. It's not too big of a deal though

- You're right, probably best to make it remove the blindness on tie (dont think that was intentional when i added it).

**Comment by ATH1909** [comment]
> * I would like that reverted.
> * Even if you're unconscious you'll still be able to roll hence why it only has qdel checks. It's not too big of a deal though
> * You're right, probably best to make it remove the blindness on tie (dont think that was intentional when i added it).

* That chance increase was literally the entire(/main) point of this PR. All of this will have been for nothing if no one will ever actually SEE that janken effect.
* Aye, that's intentional. The idea is that the effect can only trigger if you're helpless and probably have no way out other than by accepting the deal, so it can only trigger if you're NOT conscious (so you're either unconscious or in crit), which means that you're probably blind ANYWAY. Plus, even if you COULD be conscious when the prompt appears, we're already sticking a big-ass prompt in the patient's face; I don't see why we need to flat-out blind them (especially because they're probably injured because they got into/still are in combat) too.



> The RPS being compared to a pulse rifle is a bit misleading because yes, if you took 5 rolls of helbital and 5 rolls of pulse rifle, you'd have the same probability. The catch is helbital rolls every life and pulse rifle takes several minutes to even roll.

The thing is, since helbital was been added to the game, I can almost ASSURE you that more arcade games have been won than helbital ticks have occurred under the required conditions. NOBODY makes or uses helbital. It's not a roundstart chem, it deals oxyloss damage to people in softcrit, it can't heal someone up to full (brute) health, and salicylic acid does its gimmick better anyway. I've been hanging around medbay a lot as a chaplain for the past month or two, and I have literally never seen ANYONE make even so much as a SINGLE UNIT of helbital. 

I guarantee you that the number of people who have gotten the helbital janken achievement since the addition of helbital to the game is less than 1/3 (because you only get a 1/3 chance to actually win the game of RPS, even when you manage to roll that one in a million chance) of the number of people who've gotten the pulse rifle arcade machine achievement since helbital was added to the game, even accounting for people who got the pulse rifle arcade machine achievement before helbital was added to the game and thus couldn't have gotten it again after helbital was added to the game. I'd be surprised if even a SINGLE PERSON who didn't specifically grind that achievement on a private server has that janken achievement.

> I have no plans on making this chance smaller, it is suppose to practically never happen.

There's a difference between "a cool effect that rarely happens" and "a cool effect that will literally never get seen before helbital gets axed by yet another Medbay rework 3 years from now".

**Comment by ExcessiveUseOfCobblestone** [comment]
Im not interested in raising the probability, it's meant to pretty much never happen and be a super rare joke effect like getting an OP weapon from the arcade. I don't mind letting you roll for it while in softcrit though (indirectly raising the chance to see the effect).

If that's a crucial point of the PR feel free to close.

**Comment by ExcessiveUseOfCobblestone** [comment]
looks like it is

**Pull request closed without merging**